transportation.sk

```diff
@@ -0,0 +1,31 @@
+boat:
+	minecraft version = 1.8
+	[oak ]boat¦s = minecraft:boat
+
+boats:
+	minecraft version = 1.9 to 1.12
+	oak boat¦s = minecraft:boat
+	acacia boat¦s = minecraft:boat_acacia
+	dark oak boat¦s = minecraft:boat_dark_oak
+	jungle boat¦s = minecraft:boat_jungle
+	spruce boat¦s = minecraft:boat_spruce
+	birch boat¦s = minecraft:boat_birch
+	[any] boat = oak boat, acacia boat, dark oak boat, jungle boat, spruce boat, birch boat
+
+boats new:
+	minecraft version = 1.13 or newer
+	oak boat¦s = minecraft:oak_boat
+	acacia boat¦s = minecraft:acacia_boat
+	dark oak boat¦s = minecraft:dark_oak_boat
+	jungle boat¦s = minecraft:jungle_boat
+	spruce boat¦s = minecraft:spruce_boat
+	birch boat¦s = minecraft:birch_boat
+	[any] boat = oak boat, acacia boat, dark oak boat, jungle boat, spruce boat, birch boat
+
+elytra:
+	minecraft version = 1.9 or newer
+	elytra¦s = minecraft:elytra
+
+misc transport:
+	carrot[s] on [a] stick¦s = minecraft:carrot_on_a_stick
+	saddle[s] = minecraft:saddle
```

Comment:
This file should be removed, transportation aliases are being handled in a separate PR.

Reply:
Rogers that.. I will remove that